FlorinPosea/teme/3.5.react-rgb-color-game/src/App.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import ColorSquare from "./components/color-square/ColorSquare";
+import "./App.css";
+
+const DEFAULT_SQUARES = 6;
+var winningcolor = "";
```

Comment:
winningColor ar trebuii sa fie state, la fel si msg.
Si sa nu folosesti var. Motivul [aici ](https://javascript.plainenglish.io/4-reasons-why-var-is-considered-obsolete-in-modern-javascript-a30296b5f08f#:~:text=Scoping%20%E2%80%94%20the%20main%20reason%20to%20avoid%20var&text=var%20variables%20are%20scoped%20to,enclosing%20block%20denoted%20by%20%7B%20%7D%20.)